src/main/java/com/libraryman_api/borrowing/BorrowingService.java

```diff
@@ -66,9 +68,14 @@ public BorrowingService(BorrowingRepository borrowingRepository, FineRepository
      *
      * @param pageable the pagination information, including the page number and size
      * @return a {@link Page} of {@link Borrowings} representing all borrowings
+     * @throws InvalidSortFieldException if an invalid sortBy field is specified
      */
     public Page<Borrowings> getAllBorrowings(Pageable pageable) {
```

Comment:
What sorting parameters can be passed here?

Reply:
For **Books**, any field in the `Book` model (e.g., title, publisher, author, etc.) can be used for sorting.